content/300-guides/100-performance-and-optimization/150-connection-management/index.mdx

```diff
@@ -0,0 +1,286 @@
+---
+title: Connection management
+metaTitle: Connection management (Guide)
+metaDescription: 'How to approach connection management with Prisma Client in serverless and PaaS applications.'
+tocDepth: 2
+---
+
+<TopBlock>
+
+Databases can handle a limited number of concurrent connections. Each connection requires RAM, which means that simply increasing the connection limit without scaling the database:
+
+* ✔ might allow more processes to connect _but_
+* ✘ significantly affects database **performance**
```

Comment:
Are we sure that is the only thing that can happen? Could it not also just crash the database and similar?

Reply:
The OS will shut down a database with an 'out of memory' exception, I'll add that